docker-compose.yml

```
@@ -56,4 +56,22 @@ services:
+    volumes:
+      - ${PROMETHEUS_CONFIG}:/etc/prometheus/prometheus.yml
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+
+  grafana:
+    image: grafana/grafana-oss:latest
+    container_name: danji-grafana
+    ports:
+      - "3001:3000"
+    depends_on:
+      - prometheus
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)